CONTRIBUTING.md

```diff
@@ -113,6 +113,8 @@ We support development in the Eclipse and IntelliJ IDEs.
 For Eclipse, the minimum version that we support is [4.13][eclipse]. 
 For IntelliJ, the minimum version that we support is [IntelliJ 2017.2][intellij].
 
+[Docker](https://docs.docker.com/install/) is a necessary dependency for building elasticsearch. Make sure docker is installed prior to running an instance of elasticsearch from source code.
```

Comment:
@ejmiers thanks for confirming!

Regarding the change, how about the following?

```suggestion
[Docker](https://docs.docker.com/install/) is a necessary dependency for building Elasticsearch. You can run Elasticsearch without building all the artifacts with:

    ./gradlew :run
```